js_modules/dagster-ui/packages/ui-core/src/selection/SelectionInputAutoCompleteResults.tsx

```diff
@@ -194,16 +195,25 @@ const SuggestionItem = ({suggestion}: {suggestion: Suggestion}) => {
 
     icon = attributeToIcon[suggestion.displayText as Attribute];
   } else if (suggestion.type === 'attribute-with-value') {
-    const firstColon = suggestion.displayText.indexOf(':');
-    const attributeKey = suggestion.displayText.slice(0, firstColon);
-    const attributeValue = suggestion.displayText.slice(firstColon + 1);
-    label = (
-      <Box flex={{direction: 'row', alignItems: 'center', gap: 2}}>
-        <MonoSmall color={Colors.textLight()}>{attributeKey}:</MonoSmall>
-        <MonoSmall>{attributeValue}</MonoSmall>
-      </Box>
-    );
-    value = null;
+    // hacky approach to extract the substring filter for now..., upcoming refactor will have more robust logic.
+    const substringMatch = /^([a-zA-Z]+)_substring:(.+)$/.exec(suggestion.text);
```

Comment:
Sounds good!  Also I'm sure we pulled this `_substring` convention from somewhere, but `key:"a" and key_contains:"a"` feel more natural to me than substring.

Reply:
its going away soon in favor of using `*` for wildcard matching in text!